components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -114,6 +115,16 @@
  */
 @property(nonatomic, readonly) BOOL contentReachesFullscreen;
 
+/**
+ The drawer's top shadow color. Defaults to black with 20% opacity.
+ */
+@property(nonatomic, strong, nullable) UIColor *drawerShadowColor;
```

Comment:
Should this be `nonnull`? Does setting it to `nil` mean there is no shadow? That could be supported by `UIColor.clearColor`.

Reply:
Done.